src/app/views/profile/trophies/_nav/nav.vue

```diff
@@ -37,45 +37,19 @@
 		</ul>
 		<template v-if="hasGames">
 			<hr />
-			<ul class="sans-margin">
-				<li v-for="game of games" :key="game.id">
-					<router-link
-						:to="{
-							name: 'profile.trophies.game',
-							params: {
-								id: game.id,
-							},
-						}"
-						active-class="active"
-						class="-nav-item-game"
-					>
-						{{ game.title }}
-						<span class="badge -game-counter">
-							{{ game.trophyCount | number }}
-							<span v-if="gameHasUnviewedTrophies(game.id)" class="-new-notice">
-								<app-jolticon icon="exclamation-circle" notice />
-							</span>
+			<app-list-group-selector :items="games" :current="currentGame" @change="changeGame($event)">
+				<template v-slot="{ item }">
+					<translate v-if="!item">Choose a game...</translate>
+					<template v-else>
+						<span class="badge" :class="{ 'badge-notice': gameHasUnviewedTrophies(item.id) }">
+							{{ item.trophyCount | number }}
 						</span>
-					</router-link>
-				</li>
-			</ul>
+						{{ item.title }}
```

Comment:
This needs to be shortened if no space, otherwise it'll look like this:  
![image](https://user-images.githubusercontent.com/2119926/68115994-045d1900-fefa-11e9-9e48-cc506d297915.png)

Reply:
This is kind of an interesting issue. We probably need to fix in the list-group styling, since it's all within that component. Maybe try something with overflow-wrap? https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap

Reply:
I'm cool with this for now, people usually put a space in their name somewhere haha